NASA-AMMOS/aerie-ui
# Expansion permissions

resolves #723 

To test expansion rules:
1. Create a few expansion rules, each with a different user
2. Log in with one of the users used to create an expansion rule in step 1 and switch the role to `user`
3. Verify that only the expansion rules that the user owns is editable/deletable
4. Edit one of the expansion rules
5. Verify that the model, command dictionary, and activity type dropdowns are no longer editable unless the role selected is `admin`
